### DISCUSSION TITLE
Rewrites as Next.js

### DISCUSSION THREAD

**Discussion by adityar15** [nuxt/nuxt#16935] [General]
Hi, is there a feature in Nuxt 3 which is similar to Next.js [Rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites)?

**Comment by eMeRiKa13** [comment]
I'm really interested to have an answer for this question as I'm looking for that as well.

Thanks

**Comment by danielroe** [comment]
You can implement this pretty straightforwardly within Nitro: https://stackblitz.com/edit/github-mk4pd1.

If you are wanting to do this within the Vue part of your app, you should use `redirect` instead (within `definePageMeta`) so your app hydrates correctly.

**Reply by eMeRiKa13** [reply]
Thanks for your answer @danielroe.

It displays a server route? Is-it possible to display a page of my app?

**Reply by danielroe** [reply]
Of course. But as I said, you should take care your app hydrates properly, and the way to do that within the Vue part of your app is use `redirect` within `definePageMeta`.

**Reply by eMeRiKa13** [reply]
My app has multiple domain names, for a specific domain www.test.com/* I want to display www.myapp/specificpage.

With a global middleware I achieved to redirect www.test.com/* to www.test.com/specificpage but It will be better if I can just rewrite a route based on the domain.
